kubernetes/config/common.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+data:
+  common.libsonnet: |
+    {
+      blobstore: {
+        contentAddressableStorage: {
+          sharding: {
+            hashInitialization: 11946695773637837490,
+            shards: [
```

Comment:
I'm wondering, if the shards need to be 3, because the `storage` deployment has 3 replicas.

Reply:
I hadn't noticed this, thanks! I have reduced the storage replicas to 2 as this matches the docker-compose setup.